docs/installing/compiler.md

```diff
@@ -21,53 +21,60 @@ To obtain the newest package:
 
+Variant A: Using the MinGW GCC compiler toolchain
 - Download and install any MinGW GCC compiler (e.g. [7.3.0-i686-posix-dwarf](https://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/7.3.0/threads-posix/dwarf/i686-7.3.0-release-posix-dwarf-rt_v5-rev0.7z/download)) if you don't have any already in your system (MSVC and others will work too).
 - Make sure the directory containing the GCC executable is in the system PATH.
+
+Variant B: Using MSVC
+- Install Visual Studio. At the very least the VisualStudio Build Tools. The VSCode extensions will discover what you need from them automatically.
+
+For both variants:
 - Download and install [CMake](https://cmake.org) if you haven't already.
   Be sure its executable directory is in your PATH or you'll have to configure vscode extension with it manually.
-- In VSCode, install `twxs.cmake` and `ms-vscode.cmake-tools` extensions for CMake support.
 - If CMake's executable is not in your system's PATH, enter VSCode settings and set full path to cmake.exe.
-- Clone the [AmigaCMakeCrossToolchains](https://github.com/AmigaPorts/AmigaCMakeCrossToolchains) repo.
-  Be sure you have `m68k-bartman.cmake` file there.
-- Confiure VSCode CMake extension:
+- In VSCode, install `twxs.cmake` and `ms-vscode.cmake-tools` extensions for CMake support.
+- Configure VSCode CMake extension:
   - In the IDE, open any directory which will contain your project.
-  - Create empty CMakeLists.txt file **and restart the editor** so that it can discover that you're inside CMake-based project.
-  - Enter command palette (<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>) and start typing "CMake Scan for kits" and select it from the options.
-  - Enter command palette again and select "CMake Edit User-Local CMake Kits"
-  - You will see a .json file which holds the array of configurations of compilers in your system.
-    There will be one matching the MinGW compiler, but you need to append the configuration for Bartman suite manually.
-    Simply add a comma after last compiler config in the file and add the following, replacing `YOUR_USER_NAME`, `m68k-bartman.cmake` path and extension name/version so that it matches your filesystem:
-
-    ```json5
-    [
-      {
-        "name": "Some other compiler which was automatically detected",
-        // Some other fields...
+  - Clone the [AmigaCMakeCrossToolchains](https://github.com/AmigaPorts/AmigaCMakeCrossToolchains) repo next to that folder.
+    Be sure that the `m68k-bartman.cmake` file is there. Do not add the AmigaCMakeCrossToolchains to your workspace, it just
+    needs to be next to the project folder.
+  - Create a folder .vscode in your project
+  - Create a file .vscode/cmake-kits.json with the following content. *IMPORTANT:* If you use the MingGW toolchain, choose `MinGW Makefiles` as preferred generator, for MSVC choose `Ninja`.
```

Comment:
I haven't really tested ACE's cmake against ninja, especially asset conversion functions, but we'll see if there will be issues related to it, I guess. ;)

Also, I'm not sure if I'm comfortable with directly asking ppl to manually create the json file. At least there should be hint that it can be created using "Scan for kits" command and edited with "Edit kits" command.

Reply:
Ohhhhhhh, now I get it! Of course that's acceptable, but I dunno if it's good to have a windows-centric json in project repo if you e.g. share a project with linux/mac users (Bartman's toolchain started to support those too!). Of course it's an option, but I think the guide should at least hint at ability of setting up the toolchain on the local machine for use across all projects.